pytradfri/group.py

```diff
@@ -40,6 +40,8 @@ def path(self):
     @property
     def state(self):
         """Boolean representing the light state of the group."""
+        if not self.raw:
```

Comment:
I don't think this is a correct fix. We rely on the type of `raw` in many places in the code. We need to fix the problem at its source.

Reply:
I agree that it is not solving the problem. But without this patch I need to restart a couple of times each day, and with the patch it has so far been running for around 30hours.

I believe the problem is in aiocoap, but that is not in our scope, and it seems they are very slow in making releases.

Reply:
I think I am able to solve this where self.raw is assigned, which is a far better solution.

```
    def update(self):
        """
        Update the info.

        Returns a Command.
        """

        def process_result(result):
            self.raw = result

        return Command("get", self.path, process_result=process_result)
```

It is clear that Command fails from time to time (especially with many observe calls), and the library do not expect that.

Reply:
I have now implemented the check where self.raw is assigned, I hope you like this patch more.